lib/cc/analyzer/container.rb

```diff
@@ -78,9 +78,6 @@ def run(options = [])
         if @timed_out
           t_out.kill if t_out
           t_err.kill if t_err
```

Comment:
WDYT about bring this out from under the timed-out conditional? Presumably, if we're exiting this method (successfully or otherwise) we're done with these threads. If we were successful we'll have joined them already and the kill should be a no-op -- if not, better kill-than-sorry...